src/box/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ lua_source(lua_sources lua/config/utils/textutils.lua      config_utils_textutil
 if (ENABLE_CONFIG_EXTRAS)
     lua_source(lua_sources ${CONFIG_EXTRAS_DIR}/source/etcd.lua config_source_etcd_lua)
     lua_source(lua_sources ${CONFIG_EXTRAS_DIR}/storage/init.lua config_storage_init_lua)
+    lua_source(lua_sources ${CONFIG_EXTRAS_DIR}/storage_client.lua config_storage_client_lua)
     lua_source(lua_sources ${CONFIG_EXTRAS_DIR}/source/storage.lua config_source_storage_lua)
     lua_source(lua_sources ${CONFIG_EXTRAS_DIR}/extras.lua config_extras_lua)
 endif()
```

Comment:
Maybe it is the good time to finally move the list of these EE config modules to the EE sources? See `ENABLE_FAILOVER` below for example. It is tracked in TNTP-280.

Reply:
After discussing, I've changed this PR purpose to migrate extras to EE.
Thus, the comment is applied.
Please, take a look now.